src/api/app/controllers/webui/subscriptions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class Webui::SubscriptionsController < Webui::WebuiController
+  before_action :require_admin
+
+  def index
+    @subscriptions_form = subscriptions_form
+  end
+
+  def update
+    subscriptions_form.update!(params[:subscriptions])
```

Comment:
It will go wrong. Someone will add a validation to `EventSubscription` or something else you did not plan for.